static/app/components/feedback/feedbackItem/screenshotsModal.tsx

```diff
@@ -0,0 +1,88 @@
+import {Fragment, useState} from 'react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import type {ModalRenderProps} from 'sentry/actionCreators/modal';
+import ScreenshotPagination from 'sentry/components/events/eventTagsAndScreenshot/screenshot/screenshotPagination';
+import FeedbackScreenshot from 'sentry/components/feedback/feedbackItem/feedbackScreenshot';
+import PanelHeader from 'sentry/components/panels/panelHeader';
+import {t, tct} from 'sentry/locale';
+import type {EventAttachment, Organization, Project} from 'sentry/types';
+
+type Props = ModalRenderProps & {
+  initialIndex: number;
+  organization: Organization;
+  projectSlug: Project['slug'];
+  screenshots: EventAttachment[];
+};
+
+export default function ScreenshotsModal({
+  Body,
+  Header,
+  initialIndex,
+  organization,
+  projectSlug,
+  screenshots,
+}: Props) {
+  // Selected index can be any integer, positive or negative
+  const [selectedIndex, setSelectedIndex] = useState(initialIndex);
+  // Current index maps the selectedIndex into the range for which we have screenshots
+  const currentIndex = selectedIndex % screenshots.length;
+  // Cannot be undefined because of the modulo above:
+  const currentScreenshot = screenshots.at(currentIndex)!;
+
+  return (
+    <Fragment>
+      <Header closeButton>
+        {screenshots.length ? (
```

Comment:
<img width="1343" alt="SCR-20240402-pcan" src="https://github.com/getsentry/sentry/assets/56095982/ed6b7dca-6e7a-4025-b356-ea8385fb8291">
looks like if there's only 1 screenshot, we'll show `1 of 1` instead of `Screenshot` (which is fine, but then the ternary isn't used)

-- slightly related & small detail, but the next/forward buttons aren't disabled in this case which could lead to dead clicks!

Reply:
Dead clicks! will fix.